src/test/java/com/wikia/webdriver/common/core/networktrafficinterceptor/NetworkTrafficInterceptor.java

```diff
@@ -1,40 +1,30 @@
 package com.wikia.webdriver.common.core.networktrafficinterceptor;
 
-import java.util.HashMap;
-import java.util.Map;
+import java.net.InetSocketAddress;
+import java.util.concurrent.TimeUnit;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.concurrent.TimeUnit'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)